lib/manageiq/appliance_console/message_configuration.rb

```diff
@@ -187,12 +187,6 @@ def host_reachable?(host, what)
         true
       end
 
-      def configure_messaging_type(value)
-        say(__method__.to_s.tr("_", " ").titleize)
-
-        ManageIQ::ApplianceConsole::Utilities.rake_run!("evm:settings:set", ["/prototype/messaging_type=#{value}"])
-      end
-
```

Comment:
NOTE this will be the default (once https://github.com/ManageIQ/manageiq/pull/21997 is merged) and will not need to be set by appliance console